config/vault.go

```diff
@@ -178,7 +181,17 @@ func (c *VaultConfig) Finalize() {
 	c.SSL.Finalize()
 
 	if c.Token == nil {
-		c.Token = String("")
+		// Use the vault CLI's token, if present.
+		homePath, _ := homedir.Dir()
```

Comment:
Hmm. I'm very hesitant to discard the errors like this. A better approach might be to do this in the CLI.